source/extensions/filters/network/http_connection_manager/config.cc

```diff
@@ -296,11 +299,13 @@ HttpConnectionManagerConfig::HttpConnectionManagerConfig(
           PROTOBUF_GET_OPTIONAL_MS(config.common_http_protocol_options(), max_connection_duration)),
       max_stream_duration_(
           PROTOBUF_GET_OPTIONAL_MS(config.common_http_protocol_options(), max_stream_duration)),
-      stream_idle_timeout_(
-          PROTOBUF_GET_MS_OR_DEFAULT(config, stream_idle_timeout, StreamIdleTimeoutMs)),
+      stream_idle_timeout_(PROTOBUF_GET_MS_OR_DEFAULT(
+          config, stream_idle_timeout,
+          DefaultsProfileSingleton::get().httpConnectionManager.stream_idle_timeout)),
       request_timeout_(PROTOBUF_GET_MS_OR_DEFAULT(config, request_timeout, RequestTimeoutMs)),
-      request_headers_timeout_(
-          PROTOBUF_GET_MS_OR_DEFAULT(config, request_headers_timeout, RequestHeaderTimeoutMs)),
```

Comment:
The non used variables (e.g., `RequestHeaderTimeoutMs`) should also be removed (I think it is defined in the `config.h` file).

Reply:
It is still needed in case if profile does not have a corresponding value or defaults profile was not specified.